exercises/J-array-get-set/exercises2.js

```diff
@@ -7,7 +7,7 @@
 */
```

Comment:
Here you missed the second request - change the first value in the array to the number 1